baselines/Gmail Minimum Viable Secure Configuration Baseline v0.2.md

```diff
@@ -723,18 +713,15 @@ POP and IMAP access MAY be enabled on a per-user and per-application basis as ne
 
 To configure the settings for POP and IMAP access:
 
-#### Policies Group 9 common Instructions
+
+#### GWS.GMAIL.9.1v0.2 Instructions
 1.  Sign in to the [Google Admin Console](https://admin.google.com).
 2.  Select **Apps -\> Google Workspace -\> Gmail**.
 3.  Select **End User Access -\> POP and IMAP access**.
 4.  Follow the implementation for each policy.
```

Comment:
"Follow the implementation for each policy" needs to be cut as there's only one policy now, and the steps following this one renumbered.